client/hooks/useEndpointAction.js

```diff
@@ -24,3 +24,25 @@ export const useEndpointAction = (httpMethod, endpoint, params = {}, successMess
 		}
 	}, [dispatchToastMessage, params, sendData, successMessage]);
 };
+
+export const useEndpointActionExperimental = (httpMethod, endpoint, successMessage) => {
```

Comment:
Maybe I missing something... In all use cases I've seen for this hook there was no reason to `params` to be called at the callback. E.g.:

```
const fn = useEndpointActionExperimental('POST', '/xyz');
const fnAction = useMutableCallback((...args) => {
  fn(params, ...args);
});
```

When this would be better:

```
const fnAction = useEndpointAction('POST', '/xyz', params);
```